src/transport/tcp/TCPConnection.h

```diff
@@ -152,6 +152,7 @@ class INET_API TCPStateVariables : public cObject
     TCPStateVariables();
     virtual std::string info() const;
     virtual std::string detailedInfo() const;
+    virtual void reInitialState() {};
```

Comment:
Would be better write a `virtual void setSendQueueLimit(uint32 newLimit) { sendQueueLimit = newLimit; }` function,
use it in `TCPConnection::process_QUEUE_BYTES_LIMIT()`,
and overwrite `setSendQueueLimit` in `TCPTahoeRenoFamilyStateVariables` class for update `sstresh`, too.